### PR TITLE
WRO-4836: Add pinch zoom sampler

### PR DESCRIPTION
**Pull request by jeonghee27**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
New feature added on @enact/ui/touchable to support pinch Zoom gesture 

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Add pinch zoom sampler

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Multi-touch cannot be tested in PC Env.
In oder to test, Add a test data to ui/Touch.js getEventCoordinates L24

```
		} else {
			x = ev.targetTouches[0].clientX;
			y = ev.targetTouches[0].clientY;
			// TEST data
			return [{x: 400, y: 400}, {x, y}];
		}
```
### Links
[//]: # (Related issues, references)
WRO-4836

### Comments
Enact-DCO-1.0-Signed-off-by: Jeonghee Ahn (jeonghee27.ahn@lge.com)

### CODE REVIEW

File: samples/sampler/stories/qa/Touchable.js [hunk 1]
```diff
@@ -210,6 +211,45 @@ boolean('disabled', WithDragHandlers, TouchableDiv);
...
+				maxZoom: args['pinchZoomConfig maxZoom'],
+				minZoom: args['pinchZoomConfig minZoom'],
+				scaleTolerance: args['pinchZoomConfig scaleTolerance']
+			}}
+			onPinchZoomStart={action('onPinchZoomStart')}
+			onPinchZoom={handlePinchZoom}
+			onPinchZoomEnd={action('onPinchZoomEnd')}
+			style={{
+				border: '4px dashed #888',
+				margin: ri.unit(ri.scale(300), 'rem'),
```
Comment (0x64) [inline samples/sampler/stories/qa/Touchable.js:235]:
```suggestion
				margin: ri.scaleToRem(300),
```

Comment (jeonghee27) [reply samples/sampler/stories/qa/Touchable.js]: I fixed

File: samples/sampler/stories/qa/Touchable.js [hunk 2]
```diff
@@ -210,6 +211,45 @@ boolean('disabled', WithDragHandlers, TouchableDiv);
...
+		setScale(ev.scale);
+	}, []);
+
+	return (
+		<TouchableDiv
+			pinchZoomConfig={{
+				global: args['pinchZoomConfig global'] || false,
+				maxZoom: args['pinchZoomConfig maxZoom'],
+				minZoom: args['pinchZoomConfig minZoom'],
+				scaleTolerance: args['pinchZoomConfig scaleTolerance']
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/Touchable.js:228]: Shouldn't this be `moveTolerance`?

**Review by MikyungKim — APPROVED** [review]
LGTM

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**